core/common/src/main/java/alluxio/util/CommonUtils.java

```diff
@@ -280,5 +283,23 @@ public static boolean isUfsObjectStorage(String ufsAddress) {
         || ufsAddress.startsWith(Constants.HEADER_OSS);
   }
 
+  /**
+   * Gets the value from the static key=value mapping in string format. E.g. mapping
+   * "id1=user1;id2=user2" returns value "user1" with key "id1", returns value null with key
+   * "invalid". Returns null if the given key does not exist in the mapping.
+   *
+   * @param mapping the "key=value" mapping in string format separated by ";"
```

Comment:
Good point, I can make it more general by specifying separator `;` and connector `=` as function parameters. I think it still makes sense to be CommonUtils because I can foresee new mapping-like configuration would use this to parse key value pairs. It's arguable that we can make the convention of such mapping to use `key1=value1;key2=value2`
What do you think?